test/server/cards/04.2-CtA/SummerHarvest.spec.js

```diff
@@ -26,22 +26,6 @@ describe('SummerHarvest', function() {
             it('should calculate the gold amount properly', function() {
                 expect(this.summerHarvest.getIncome()).toBe(7);
             });
-
-            describe('when playing it again after going through all plots', function() {
-                beforeEach(function() {
-                    this.completeMarshalPhase();
-                    this.completeChallengesPhase();
-
-                    // Move Summer Harvest back to the plot deck so it's eligible to be picked again.
-                    this.summerHarvest.controller.moveCard(this.summerHarvest, 'plot deck');
-
-                    this.selectFirstPlayer(this.player1);
-                });
-
-                it('should still properly calculate the gold amount properly', function() {
-                    expect(this.summerHarvest.getIncome()).toBe(7);
```

Comment:
This test was very strange and inconsistent:
* The deck only has one plot in it, so it's unclear if we have defined behaviour for that after recycling.
* To get around that, `moveCard` was called manually instead of the more appropriate `dragCard` to simulate UI interaction.
* The assertion itself was wrong - this ends up being Summer Harvest vs Varys's Riddle, and given all the above it wasn't prompting properly for that interaction and instead just set income to 7.

Since the bug it was protecting against is long gone - in the original version, it was adding to the plot's gold modifier directly each round instead of setting a base income - it didn't seem worth the trouble to rework the test.